Sources/Subs.php

```diff
@@ -1752,15 +1733,20 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 					'width' => array('optional' => true, 'value' => ' width="$1"', 'match' => '(\d+)'),
 					'height' => array('optional' => true, 'value' => ' height="$1"', 'match' => '(\d+)'),
 				),
-				'content' => '<img src="$1" alt="{alt}" title="{title}"{width}{height} class="bbc_img resized" loading="lazy">',
-				'validate' => function(&$tag, &$data, $disabled)
+				'content' => '$1',
+				'validate' => function(&$tag, &$data, $disabled, $params)
 				{
-					$data = strtr($data, array('<br>' => ''));
+					$url = strtr($data, array('<br>' => ''));
 
-					if (parse_url($data, PHP_URL_SCHEME) === null)
-						$data = '//' . ltrim($data, ':/');
+					if (parse_url($url, PHP_URL_SCHEME) === null)
+						$url = '//' . ltrim($url, ':/');
 					else
-						$data = get_proxied_url($data);
+						$url = get_proxied_url($url);
+
+					$alt = !empty($params['{alt}']) ? ' alt="' . $params['{alt}']. '"' : '';
+					$title = !empty($params['{title}']) ? ' title="' . $params['{title}']. '"' : '';
+
+					$data = '<img src="' . $url . '"' . $alt . $title . $params['{width}'] . $params['{height}'] . 'class="bbc_img' . (!empty($params['{width}']) || !empty($params['{height}']) ? ' resized' : '') . '" loading="lazy">';
```

Comment:
Hm. If this BBC is disabled, `$data` should be set to just the URL. Otherwise, the `disabled_content` string will still replace `$1` with the `<img>` element in the output.
```suggestion
					$data = in_array($tag['tag'], $disabled) ? $url : '<img src="' . $url . '"' . $alt . $title . $params['{width}'] . $params['{height}'] . 'class="bbc_img' . (!empty($params['{width}']) || !empty($params['{height}']) ? ' resized' : '') . '" loading="lazy">';
```

Reply:
Won't the 'disabled_content' => '($1)', do that?